### PR TITLE
[Admin] Price History Navigation from Product Variant Index and Edit Pages

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A
|-----------------|-----
| Branch?         | 2.0 
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no 
| Related tickets | fixes #X, partially #Y, mentioned in #Z
| License         | MIT

### CODE REVIEW

**Review by mpysiak — DISMISSED** [review]

**Review by Wojdylak — DISMISSED** [review]

File: src/Sylius/Bundle/AdminBundle/templates/product_variant/grid/action/price_history.html.twig [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{% from '@SyliusAdmin/shared/helper/icon.html.twig' import icon %}
+
+{% set visible = options.visible is defined ? options.visible : true %}
+
+{% if visible %}
+    <div>
+        <span class="dropdown-header">{{ action.label|trans }}</span>
+        {% for channelPricing in data.channelPricings %}
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/templates/product_variant/grid/action/price_history.html.twig:8]:
```suggestion
        {% for channel_pricing in data.channelPricings %}
```

File: src/Sylius/Bundle/AdminBundle/templates/product_variant/grid/action/price_history.html.twig [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            <a
+                class="dropdown-item"
+                href="{{ path('sylius_admin_channel_pricing_log_entry_index', parameters) }}"
+                {{ sylius_test_html_attribute('price-history-in-channel', channelPricing.channelCode) }}
+            >
+                {{ icon({ icon: action.icon|default('eye'), class: 'icon icon-tabler' }) }}
+                <span>
+                    <span class="text-nowrap">{{ channelPricing.channelCode|sylius_channel_name }}</span>
+                </span>
+
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/templates/product_variant/grid/action/price_history.html.twig:19]:
```suggestion
```

**Review by TheMilek — DISMISSED** [review]

File: src/Sylius/Bundle/AdminBundle/templates/product_variant/form/content/update/header/actions/price_history.html.twig [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{% from '@SyliusAdmin/shared/helper/icon.html.twig' import icon %}
+
+{% set product_variant = hookable_metadata.context.resource %}
+
+{% if product_variant.channelPricings|length < 1 %}
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/templates/product_variant/form/content/update/header/actions/price_history.html.twig:5]: I guess it should be 🤔:
```suggestion
{% if product_variant.channelPricings|length == 1 %}
```
or
```suggestion
{% if product_variant.channelPricings|length < 2 %}
```

Comment (Rafikooo) [reply src/Sylius/Bundle/AdminBundle/templates/product_variant/form/content/update/header/actions/price_history.html.twig:5]: I revised the logic slightly to display the list even if there is only one channel. For fewer than one channel, I disabled the button.

**Review by GSadee — APPROVED** [review]

**Pull request merged**